lib/parse/credit.js

```diff
@@ -18,3 +18,8 @@ export const calculateReliability = (promises) =>
     .map((p) => p.credit)
     .compact()
     .mean()
+
+// Return the count of promises included in the reliability calculation.
+// Right now, that is all promises completed or overdue - "due or done"
+export const promisesIncluded = (promises) => 
```

Comment:
i'm thinking it would be cleaner to modify the `calculateReliability` function to return an object e.g. `{ score: 99.9999, counted: 123 }`, and that way we also only have to call `map()` and `compact()` once per user

Reply:
Yes, makes sense.  Would also allow for future changes to the calculation (say, looking at only at most the last N promises) that might impact the calculation, and would keep all the code in one place for those.